src/image-cache.js

```diff
@@ -30,14 +30,14 @@ async function getOrUpdateCachedImage(uri, tag) {
       buffer.type = "image/png";
     } catch (err) {
       console.error("Failed to convert SVG to PNG", err);
-      return undefined;
+      return uri.slice(0, 100);
```

Comment:
What is the point of returning part of the SVG? It might be better to have nothing returned than a broken image...